data/docs/setup-alerts-notification.mdx

```diff
@@ -3,43 +3,46 @@ date: 2024-06-06
 id: setup-alerts-notification
 title: Setup Alerts Notifications Channel
 ---
-You can setup notification channel for sending the generated alerts to other applications. Currently, the following channels are supported.
-
-<DocCardContainer>
-
-<DocCard
-    title="📄️ Slack"
-    description=""
-    href="/docs/alerts-management/notification-channel/slack"
-/>
-
-<DocCard
-    title="📄️ Webhook"
-    description=""
-    href="/docs/alerts-management/notification-channel/webhook"
-/>
-
-<DocCard
-    title="📄️ PagerDuty"
-    description=""
-    href="/docs/alerts-management/notification-channel/pagerduty"
-/>
-
-<DocCard
-    title="📄️ Opsgenie"
-    description=""
-    href="/docs/alerts-management/notification-channel/opsgenie"
-/>
-
-<DocCard
-    title="📄️ MS Teams"
-    description=""
-    href="/docs/alerts-management/notification-channel/ms-teams"
-/>
-
-<DocCard
-    title="📄️ Email"
-    description=""
-    href="/docs/alerts-management/notification-channel/email"
-/>
-</DocCardContainer>
\ No newline at end of file
+---
+date: 2024-06-06
+id: setup-alerts-notification
+title: Setup Alerts Notifications Channel
+---
+## Setting a Notification Channel
+
+As a first step before creating a new alert, you should set up a notification channel where you want to send your alert. There are multiple notification channels available to send your alerts. Here's how you can set the notification channel:
+
+### Step 1: Open the Alert Channel Settings
+To configure the appropriate notification channel, navigate to the Alert Channel settings (`Settings -> Alert Channel Tab`). This section contains a list of available channels that you have created.
+
+### Step 2: Create a New Alert Channel
+Click on the **`+ New Alert Channel`** button at the top right corner and fill in the details
+
+#### Name
+Assign a descriptive name to the notification channel in the "Name" field. This helps identify the channel when selecting it for alerts.
+
+#### Send Resolved Alerts
+The "Send Resolved Alerts" toggle lets you decide whether to send notifications when alerts are resolved. By default it is enabled.
+
+#### Type
+The "Type" dropdown menu lets you choose the type of notification channel. The following types are available:
+
+- **[Slack](https://signoz.io/docs/alerts-management/notification-channel/slack/)**: Sends alert notifications to Slack. You can select a specific Slack channel for alert notifications.
```

Comment:
Instead of opening slack in new window can navigate to slack directly in same tab?

Reply:
@makeavish I don't understand this comment. Can you elaborate what you're suggesting ?